relay-event-normalization/src/normalize/span/description/mod.rs

```diff
@@ -532,6 +562,57 @@ fn scrub_function(string: &str) -> Option<String> {
     Some(FUNCTION_NORMALIZER_REGEX.replace_all(string, "*").into())
 }
 
+fn scrub_mongodb_query(query: &str, command: &str, collection: &str) -> Option<String> {
+    let mut query: Value = serde_json::from_str(query).ok()?;
+
+    let root = query.as_object_mut()?;
+
+    for value in root.values_mut() {
+        scrub_mongodb_visit_node(value, 3);
+    }
+
+    let scrubbed_collection_name =
+        if let Cow::Owned(s) = TABLE_NAME_REGEX.replace_all(collection, "{%s}") {
+            s
+        } else {
+            collection.to_owned()
+        };
```

Comment:
nit: I _think_ this is the same

```suggestion
    let scrubbed_collection_name = TABLE_NAME_REGEX.replace_all(collection, "{%s}").to_owned();
```

Reply:
The above returns the type error:

``` error[E0308]: mismatched types
   --> relay-event-normalization/src/normalize/span/description/mod.rs:575:51
    |
575 |     root.insert(command.to_owned(), Value::String(scrubbed_collection_name));
    |                                     ------------- ^^^^^^^^^^^^^^^^^^^^^^^^- help: try using a conversion method: `.to_string()`
    |                                     |             |
    |                                     |             expected `String`, found `Cow<'_, str>`
    |                                     arguments to this enum variant are incorrect
    |
    = note: expected struct `std::string::String`
                 found enum `std::borrow::Cow<'_, str>`
```

But
```rust
let scrubbed_collection_name = TABLE_NAME_REGEX.replace_all(collection, "{%s}").to_string();
```

compiles, if that seems reasonable?

Either way I'm going to merge without this change just in case, and loop back afterwards.

Reply:
Yep, that works!